examples/standalone-airgap/main.tf

```diff
@@ -0,0 +1,64 @@
+provider "aws" {
+  assume_role {
+    role_arn = var.aws_role_arn
+  }
+
+  default_tags {
+    tags = var.tags
+  }
+}
+
+# Keypair for SSH
+# ---------------
+resource "tls_private_key" "main" {
+  algorithm = "RSA"
+}
+
+resource "local_file" "private_key_pem" {
+  filename = "${path.module}/work/private-key.pem"
+
+  content         = tls_private_key.main.private_key_pem
+  file_permission = "0600"
+}
+
+resource "aws_key_pair" "main" {
+  public_key = tls_private_key.main.public_key_openssh
+
+  key_name = "${var.friendly_name_prefix}-ssh"
+}
+
+# Store TFE License as secret
+# ---------------------------
+
+module "secrets" {
+  source = "../../fixtures/secrets"
+  tfe_license = {
+    name = "${var.friendly_name_prefix}-license"
+    path = var.license_file
+  }
+}
+# Standalone, mounted disk
+# ------------------------
+module "standalone" {
+  source = "../../"
+
+  operational_mode    = var.operational_mode
```

Comment:
I think we can leave this off so the example submodules are only as prescriptive as they need to be.
```suggestion
```

Reply:
Same comment as above.